### ISSUE TITLE
IC2 Machines disappearing

### ISSUE THREAD

**Issue by blichte1** [NillerMedDild/Enigmatica2Expert#636]
Mod pack version: 1.51 and earlier.

This is a follow on from the previous other user report of this issue. I can confirm that they are disappearing without being powered. At first I had suspected that maybe it was a power compatibility issue/explosion and removed all IC2 machines from my power infrastructure (previous machines disappeared when not in use even with transformer upgrades, so this was a long shot idea anyway). Upon re-logging today, I noticed that several of the machines have now disappeared. 

As a weird question, is it possible that they are being treated as 'food' by some of the passive mobs? All of the machines that have disappeared to date have been at ground level. 

Bruce

**Comment by NillerMedDild** [comment]
Thanks for the follow-up, I should be able to report it to IC2 now - Though honestly, I doubt it will be fixed. We have very little information to give.

It's not possible, as I disabled the Animania plant-eating feature in 1.51 :)

**Comment by ADAUS-joneill** [comment]
I am seeing this too.

I've placed down the following machines on my server and in single player mode.

Compressor
Metal Former

I can confirm that they just seem to 'disapear'.

It only happens if you are NOT looking at the machines.  If you are looking at the machines, they do NOT disappear.


I am suspecting it's mob interaction, here why...


I had these machines running for days without any issues, then I got a bunch of 'random animal' spawn eggs for a quest reward.  I spawned a bunch of animals around my base for a bit of fun.

Then I noticed the mobs jumping all over the place, over machines etc... it was then that my IC2 machines began to vanish, and ONLY the IC2 machines.

Also, these mobs act strange, they move around.. then 'warp' back to the place where I spawned them in, sometimes breaking crops if their 'path' back to where I spawned them passed through a crop.

I suspect (it's a long shot), that the IC2 machines are being destroyed by these mobs.

I will kill these poor things, and see if the disappearing IC2 machines goes away.

**Comment by NillerMedDild** [comment]
That's not it @ADAUS-joneill, the modpack doesn't work like that.

**Comment by ADAUS-joneill** [comment]
I would agree…

But...I am leaning back towards my 'mob' theory now.

I killed off the mobs I spawned inside my little base area and in the days since (server running 24x7 and chunk loaded), not one single IC2 machine has disappeared.

As an experiment, I summoned some random mobs, one was a draft horse, after a while... the IC2 machines began to disappear.  I buried an IC2 machine inside some cobblestone.. it was untouched.

It's not much to go on i'm afraid, but I thought i'd mentioned it anyway.
